### ISSUE TITLE
Make profile default page for non-admin users

### ISSUE THREAD

**Issue by jzongker** [LiveChurchSolutions/ChurchAppsSupport#15]
If the user lacks people.view permission, send them to the profile page instead of the dashboard on login.

**Comment by PhotoPaul** [comment]
@jzongker  Is this what you had in mind? https://github.com/LiveChurchSolutions/ChumsApp/commit/5ccc3b14cbbc69567b45e13a6192da611f91270a

**Comment by PhotoPaul** [comment]
@anasdevmercs Is this still an open issue?

**Comment by jzongker** [comment]
No, sorry.  I just failed to close the task.  Thank you for fixing that.
